src/overlays/actors/ovl_En_Box/z_en_box.h

```diff
@@ -4,11 +4,84 @@
 #include <ultra64.h>
 #include <global.h>
 
+#define ENBOX_TREASURE_FLAG_UNK_MIN 20
+#define ENBOX_TREASURE_FLAG_UNK_MAX 32
+
 struct EnBox;
 
+typedef void (*EnBoxActionFunc)(struct EnBox*, GlobalContext*);
+
+typedef enum {
+    /*
+     set on init unless treasure flag is set
+     if clear, chest moves (Actor_MoveForward) (falls, likely)
+     ends up cleared from SWITCH_FLAG_FALL types when switch flag is set
+    */
+    ENBOX_STATE_IMMOBILE = 1 << 0,
+    /*
```

Comment:
imo for movement flags it would make more sense for the prefix to be ENBOX_MOVE_ instead of "state"